### PR TITLE
refactor(frontend): adjust form validation for asset and price manager

### PR DESCRIPTION
**Pull request by lukicenturi**
Part of #5997

## Checklist

- [x] ManagedAssetForm
- [x] CustomAssetForm
- [x] HistoricPriceForm
- [x] LatestPriceForm

### CODE REVIEW

File: frontend/app/src/pages/price-manager/latest/index.vue [hunk 1]
```diff
@@ -108,30 +115,24 @@ onMounted(async () => {
...
-      :display="showForm"
+      :display="openDialog"
       :title="
         editMode
           ? t('price_management.dialog.edit_title')
           : t('price_management.dialog.add_title')
       "
-      :action-disabled="!valid"
-      @confirm="managePrice()"
+      :action-disabled="!valid || submitting"
```
Comment (kelsos) [inline frontend/app/src/pages/price-manager/latest/index.vue:131]: So since we are validating on submit let's also remove the `!valid` check here

**Comment by kelsos** [comment]
Like the other PR if we are adjusting to use on submit validation, let's remove the disabling of the `!valid` condition and only keep submitting/loading related disabling.

File: frontend/app/src/pages/price-manager/latest/index.vue [hunk 2]
```diff
@@ -2,26 +2,34 @@
 import omit from 'lodash/omit';
 import { type Nullable } from '@/types';
 import { type ManualPrice, type ManualPriceFormPayload } from '@/types/prices';
+import { useLatestPriceForm } from '@/composables/price-manager/latest/form';
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/src/pages/price-manager/latest/index.vue]: I will add it in next PR

**Review by kelsos — APPROVED** [review]
lgtm @lukicenturi there was a warning on the ManageAsset form I had to `!!editableItem` but otherwise it looks good

**Pull request merged**